docs/jwst/ramp_fitting/reference_files.rst

```diff
@@ -6,29 +6,33 @@ electrons, and convert the results of ramp fitting back to DN.
 The read noise values are used as part of the noise estimate for
 each pixel. Both are necessary for proper computation of noise estimates.
 
+GAIN Reference Files
+^^^^^^^^^^^^^^^^^^^^
+The GAIN reference file is selected based on instrument, detector and,
+where necessary, subarray.
 
-CRDS Selection Criteria
------------------------
-The readnoise reference file is selected by instrument, detector and, where
+READNOISE Reference Files
+^^^^^^^^^^^^^^^^^^^^^^^^^
+The READNOISE reference file is selected by instrument, detector and, where
 necessary, subarray.
 
-The gain reference file is selected based on instrument, detector and,
-where necessary, subarray.
 
+Reference File Formats
+----------------------
+
+GAIN Reference Files
+^^^^^^^^^^^^^^^^^^^^
 
-Gain Reference File Format
---------------------------
 The gain reference file is a FITS file with a single IMAGE extension,
 with ``EXTNAME=SCI``, which contains a 2-D floating-point array of gain values
 (in e/DN) per pixel. The REFTYPE value is ``GAIN``.
 
 
-Read Noise Reference File Format
---------------------------------
+READNOISE Reference Files
+^^^^^^^^^^^^^^^^^^^^^^^^^
 The read noise reference file is a FITS file with a single IMAGE extension,
-with ``EXTNAME=SCI``, which contains a 2-D floating-point array of read noise 
-values per pixel. The units of the read noise should be DN and should be the 
+with ``EXTNAME=SCI``, which contains a 2-D floating-point array of read noise values
+per pixel. The units of the read noise should be electrons and should be the
 CDS (Correlated Double Sampling) read noise, i.e. the effective noise between
 any pair of non-destructive detector reads. The REFTYPE value is ``READNOISE``.
 
```

Comment:
Currently, READNOISE reference files are in units of DN; which is what the ramp fitting code expects - the code temporarily converts to electrons for noise calculations. Some older reference files are in electrons, and should have been overridden with correct units.

Reply:
The information about the Gain reference files format should be kept.